static/css/style.css

```diff
@@ -173,19 +172,19 @@ strong {
 
 /* member */
 
-.sleepy_cat { 
+.sleepy_cat {
 	font-size: 12px;
```

Comment:
@PettraV @mojcas are you guys going to fix this since you're working on members page?